Library/Formula/influxdb.rb

```diff
@@ -11,78 +12,169 @@ class Influxdb < Formula
     sha1 "693a7b3ab9a445a9e681048ecb7aba3582821fe8" => :mountain_lion
   end
 
-  depends_on "leveldb"
-  depends_on "rocksdb"
-  depends_on "autoconf" => :build
-  depends_on "protobuf" => :build
-  depends_on "bison" => :build
-  depends_on "flex" => :build
+  devel do
+    url "https://github.com/influxdb/influxdb/archive/v0.9.0-rc4.tar.gz"
+    sha1 "b4812d4d61fc4199a8aba10750a404f40bb335ad"
+  end
+
   depends_on "go" => :build
-  depends_on "gawk" => :build
   depends_on :hg => :build
 
-  def install
-    ENV["GOPATH"] = buildpath
-    Dir.chdir File.join(buildpath, "src", "github.com", "influxdb", "influxdb")
+  if build.stable?
+    depends_on "leveldb"
+    depends_on "rocksdb"
+    depends_on "autoconf" => :build
+    depends_on "protobuf" => :build
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+    depends_on "gawk" => :build
+  else
+    [ %w{github.com/bmizerany/pat    b8a35001b773c267eb260a691f4e5499a3531600},
+      %w{github.com/boltdb/bolt      a7d19d8cd50cb700babad58d0643a4bb6ac8f36c},
+      %w{github.com/BurntSushi/toml  443a628bc233f634a75bcbdd71fe5350789f1afa},
+      %w{github.com/kimor79/gollectd 1d0fc88b7c2bf0ba79021ddca2b5f5fd9cc3a5a3},
+      %w{github.com/peterh/liner     d9335eee40a45a4f5d74524c90040d6fe6013d50},
+      %w{github.com/rakyll/statik    4a16c831de16fd27a38fab90ade0cf35844a31db},
+    ].each do |source, revision|
+      go_resource source do
+        url "https://#{source}.git", :revision => revision
+      end
+    end
```

Comment:
It's annoying I realise but would rather each of these blocks were spelled out as explicit `go_resource` blocks rather than iterating through a loop.